components/dashboard/src/settings/Teams.tsx

```diff
@@ -527,6 +528,16 @@ function AllTeams() {
                 <div className="flex-grow ">
                     <h3 className="self-center">All Teams</h3>
                     <h2>Manage teams and team members.</h2>
+                    <ItemsList className="pt-2">
+                        <Item className="w-full bg-yellow-200 hover:bg-yellow-200 py-6 px-6">
```

Comment:
**issue(non-blocking)**: Added the hover state here only to bypass the redundant hover state in this case, most probably introduced in https://github.com/gitpod-io/gitpod/pull/4454.